File: SWIFT4.0/TESTS/python_SonicOS/8.2.1/Network/IP_Helper_V3_TP2169/confs/install_netcat.py
import os
from time import sleep

print('chdir /root/iphelper/')
os.chdir('/root/iphelper/')
path = os.getcwd()
print('current working directory:'.format(path))
cmd = 'tar zxvf netcat-0.7.1.tar.gz'
# print('run cmd:{}'.format(cmd))
os.system(cmd)
sleep(1)
print('chdir /root/iphelper/netcat-0.7.1')
os.chdir('/root/iphelper/netcat-0.7.1/')
sleep(1)
cmds = (
    './configure',
    'make',
    'make install'
)
for cmd in cmds:
    sleep(1)
    os.system(cmd)

File: SWIFT4.0/TESTS/python_SonicOS/8.2.1/Network/IP_Helper_V3_TP2169/confs/install_netcat.pl
#!/usr/bin/perl
print "chdir '/root/iphelper/'\n";
chdir "/root/iphelper/" or die "Cannot chdir to /root/iphelper/:$!";
sleep 1;

$cmd = "tar zxvf netcat-0.7.1.tar.gz";
print "$cmd\n";
system "$cmd";
sleep 1;

print "chdir '/root/iphelper/netcat-0.7.1'\n";
chdir "/root/iphelper/netcat-0.7.1/" or die "Cannot chdir to /root/iphelper/netcat-0.7.1:$!";
sleep 1;

$cmd = "./configure";
print "$cmd\n";
system "$cmd";
sleep 1;

$cmd = "make";
print "$cmd\n";
system "$cmd";
sleep 1;

$cmd = "make install ";
print "$cmd\n";
system "$cmd";
sleep 1;
